kalite/topic_tools/__init__.py

```diff
@@ -534,8 +534,7 @@ def smart_translate_item_data(item_data):
             elif isinstance(field_data, list):
                 item_data[field] = map(smart_translate_item_data, field_data)
 
-
-            return item_data
+        return item_data
```

Comment:
Heh, the dangers of wrong indentation. Previously, this would only iterate on a field once.